src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9401 @@
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_numParameters = 1;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_parameters[0] = Parameter::SOUND;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_numUiStrings = 2;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_uiStrings[1] = " is removed.";
+
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_internalName = "SOUND_REMOVE_ALL_DISABLED";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_uiName =
+        "Multimedia_/ Sound Events -- remove all disabled.";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_numUiStrings = 1;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_uiStrings[0] = "Remove all disabled sound events.";
+
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_ALL].m_internalName = "SOUND_ENABLE_ALL";
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_ALL].m_uiName = "Multimedia_/ Sound Events -- enable all.";
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_ALL].m_numUiStrings = 1;
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_ALL].m_uiStrings[0] = "Enable all sound events.";
+
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_internalName = "AUDIO_OVERRIDE_VOLUME_TYPE";
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_uiName =
+        "Multimedia_/ Sound Events -- override volume -- type.";
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_numParameters = 2;
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_parameters[0] = Parameter::SOUND;
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_parameters[1] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_numUiStrings = 3;
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_uiStrings[1] = " play at ";
+    m_actionTemplates[ScriptAction::AUDIO_OVERRIDE_VOLUME_TYPE].m_uiStrings[2] = "% of full volume.";
+
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_TYPE].m_internalName = "AUDIO_RESTORE_VOLUME_TYPE";
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_TYPE].m_uiName =
+        "Multimedia_/ Sound Events -- restore volume -- type.";
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_TYPE].m_numParameters = 1;
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_TYPE].m_parameters[0] = Parameter::SOUND;
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_TYPE].m_numUiStrings = 3;
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_TYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_TYPE].m_uiStrings[1] = " play at normal volume.";
+
+    m_actionTemplates[ScriptAction::AUDIO_RESTORE_VOLUME_ALL_TYPE].m_internalName = "AUDIO_RESTORE_VOLUME_ALL_TYPE";
```

Comment:
Does not set m_numParameters

Reply:
Original code does not set it (and leaves it at the default of 0)